spacy/pipeline/spancat_exclusive.py

```diff
@@ -0,0 +1,308 @@
+from typing import Callable, Dict, Iterable, List, Optional, Tuple, cast
+from dataclasses import dataclass
+
+import numpy
+from thinc.api import Config, Model
+from thinc.types import Floats2d, Ints2d, Ragged
+
+from ..language import Language
+from ..tokens import Doc, Span, SpanGroup
+from ..training import Example
+from ..vocab import Vocab
+from .spancat import spancat_score, Suggester
+from .spancat import SpanCategorizer
+
+
+spancat_exclusive_default_config = """
+[model]
+@architectures = "spacy.SpanCategorizer.v1"
+scorer = {"@layers": "Softmax.v2"}
+
+[model.scorer.init_W]
+@initializers = "zero_init.v1"
+
+[model.scorer.init_b]
+@initializers = "zero_init.v1"
+
+[model.reducer]
+@layers = spacy.mean_max_reducer.v1
+hidden_size = 128
+
+[model.tok2vec]
+@architectures = "spacy.Tok2Vec.v2"
+[model.tok2vec.embed]
+@architectures = "spacy.MultiHashEmbed.v1"
+width = 96
+rows = [5000, 2000, 1000, 1000]
+attrs = ["ORTH", "PREFIX", "SUFFIX", "SHAPE"]
+include_static_vectors = false
+
+[model.tok2vec.encode]
+@architectures = "spacy.MaxoutWindowEncoder.v2"
+width = ${model.tok2vec.embed.width}
+window_size = 1
+maxout_pieces = 3
+depth = 4
+"""
+
+DEFAULT_SPANCAT_MODEL = Config().from_str(spancat_exclusive_default_config)["model"]
```

Comment:
I think it'll be less confusing if this has a unique name:

```suggestion
DEFAULT_EXCL_SPANCAT_MODEL = Config().from_str(spancat_exclusive_default_config)["model"]
```